lib/worker.coffee

```diff
@@ -59,10 +60,7 @@ registerBlockPositions = (doc) ->
       doc.id = "__asciidoctor-preview-#{doc.$object_id()}__"
     doc.header.id = doc.id
 
-  # Use Ruby API:
-  # because `doc.findBy()` doesn't yet accept a filter function as parameter.
-  # https://github.com/asciidoctor/asciidoctor.js/issues/282
-  blocks = Opal.block_send doc, 'find_by', (b) -> b.$lineno() isnt Opal.nil
+  blocks = doc.findBy((b) -> b.$lineno() is 'number')
```

Comment:
@ldez Once https://github.com/asciidoctor/asciidoctor.js/pull/297 is merged, I think we could use the syntax `?`:

```javascript
blocks = doc.findBy((b) -> b.getLineNumber()?)
```